src/helper/remote_config/protocol/cached_target_file_hash.hpp

```diff
@@ -0,0 +1,24 @@
+// Unless explicitly stated otherwise all files in this repository are
+// dual-licensed under the Apache-2.0 License or BSD-3-Clause License.
+//
+// This product includes software developed at Datadog
+// (https://www.datadoghq.com/). Copyright 2021 Datadog, Inc.
+#pragma once
+
+#include <string>
+
+namespace dds::remote_config::protocol {
+
+class cached_target_files_hash {
+public:
+    cached_target_files_hash(std::string algorithm, std::string hash)
```

Comment:
When passing strings as function arguments, they will be copied, so you should either use `const std::string&` to avoid an extra copy (or alternatively `std::string_view`), or you can use `std::string&&` (rvalue reference) to `std::move` the original string if it's no longer needed after creating this class.

Reply:
Done